src/CopyAsHtml/CopyAsHtml/FormattedStringBuilder.cs

```diff
@@ -97,16 +98,24 @@ private void Append(string text)
             _stringBuilder.Append(text);
         }
 
-
         private IList<ClassificationSpan> GetClassificationSpansSync(SnapshotSpan parentSpan)
         {
             IAccurateClassifier accurateClassifier = _classifier as IAccurateClassifier;
 
             if (accurateClassifier != null && _waitIndicator != null)
             {
-                using (var waitContext = WaitHelper.Wait(_waitIndicator, "HTML Copy", "Formatting document for copying"))
+                try
+                {
+                    using (var waitContext = WaitHelper.Wait(_waitIndicator, "HTML Copy", "Formatting document for copying"))
```

Comment:
Do we know which component doesn't react to this cancellation token? I think I've seen a comment saying Cancel button doesn't work.

Reply:
Yes that's what worries me too. Eventually it's the taggers and the aggregate tagger and GetTags().

Another option is to drop AccurateTagger support entirely here. If a regular copy is working fine for them, this should too.

Reply:
I think @gundermanc has UI delay bug related to this, where LSP client makes remote call to get this information.
I'd suggest dropping IAccurateTagger altogether. The impact is that in this case you would get what you see, which should be good enough.

Reply:
Btw, we build PPT from an internal repo for legacy reasons

Reply:
I rewrote it to just delete IAccurateTagger